runtime/doc/lua.txt

```diff
@@ -2555,6 +2555,9 @@ vim.ui.open({path})                                            *vim.ui.open()*
     `explorer.exe`, Linux `xdg-open`, …), or returns (but does not show) an
     error message on failure.
 
+    Instead of the system default handler the function calls a command set in
+    the 'vim.g.netrw_browsex_viewer', if it is present.
+
```

Comment:
My preference would be to leave support for this `netrw` variable undocumented (we can silently support it to make people transitioning from netrw easier).

If we want to document another way to do this, I'd prefer using a different name for this variable. Perhaps this could even be a variable in the `vim.ui` module/namespace (e.g. `vim.ui.opener = 'foobar'`).

Reply:
Yes, not documenting it is one way, and of course, I haven’t expected (and I mentioned it above) that this name would survive, but I don’t know enough about naming conventions in `neovim` to suggest a better name.